apps/editing-toolkit/editing-toolkit-plugin/wpcom-domain-upsell-callout/class-wpcom-domain-upsell-callout.php

```diff
@@ -83,7 +83,22 @@ public function enqueue_script_and_style() {
 	private function should_not_show_callout() {
 		$blog_id = defined( 'IS_WPCOM' ) && IS_WPCOM ? get_current_blog_id() : \Jetpack_Options::get_option( 'id' );
 
-		return $this->blog_has_custom_domain( $blog_id ) || $this->user_has_email_unverified() || $this->blog_is_p2( $blog_id );
+		return $this->has_unlaunched_launchpad() || $this->blog_has_custom_domain( $blog_id ) || $this->user_has_email_unverified() || $this->blog_is_p2( $blog_id );
+	}
+
+	/**
+	 * Check for site-intent and not launched site on launchpad.
+	 */
+	private function has_unlaunched_launchpad() {
+		$site_intent      = defined( 'IS_WPCOM' ) && IS_WPCOM ? get_option( 'site_intent' ) : \Jetpack_Options::get_option( 'site_intent' );
```

Comment:
Whats the wpcom check here for? (I notice its not needed for the next line)

Reply:
Done!

Reply:
Looks like it's just wpcom blog id that gets stored as a jetpack option for atomic sites. 

Going by this line that popped up in grep https://github.com/Automattic/wp-calypso/blob/df14f3f4e8333002d640409e532a1623fe854036/apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/index.php#L30

Reply:
Interesting, Do you think we have do the same on the domain upsell file?

Reply:
If it's needing a wpcom blog/site id and running in an atomic context then yes.

Reply:
link?